exchanges/sharedtestvalues/sharedtestvalues.go

```diff
@@ -47,3 +55,48 @@ func NewTestWebsocket() *stream.Websocket {
 		Match:             stream.NewMatch(),
 	}
 }
+
+// SkipUnsetCredentials is a test helper function checking if the authenticated function
+// can perform the required test.
+func SkipUnsetCredentials(t *testing.T, exch exchange.IBotExchange, canManipulateOrders ...bool) {
```

Comment:
I get what you're going for making the bool optional via `...` but please throw an error if more than 1 bool present